lib/matplotlib/legend_handler.py

```diff
@@ -427,6 +440,32 @@ def create_artists(self, legend, orig_handle,
         return [legline]
 
 
+class HandlerPatchCollection(HandlerPatch):
+    """
+    Handler for `.PatchCollection` instances.
+    """
+    def _default_update_prop(self, legend_handle, orig_handle):
+        lw = _get_first(orig_handle.get_linewidths())
+        dashes = _get_first(orig_handle._us_linestyles)
+        facecolor = _first_color(orig_handle.get_facecolor())
+        edgecolor = _first_color(orig_handle.get_edgecolor())
+        legend_handle.set_facecolor(facecolor)
+        legend_handle.set_edgecolor(edgecolor)
+        legend_handle.set_linestyle(dashes)
+        legend_handle.set_linewidth(lw)
+
+    def create_artists(self, legend, orig_handle,
+                       xdescent, ydescent, width, height, fontsize, trans):
+
+        p = self._create_patch(legend, orig_handle,
+                               xdescent, ydescent, width, height, fontsize)
+
+        self.update_prop(p, orig_handle, legend)
+        p.set_transform(trans)
+
+        return [p]
```

Comment:
This appears to be identical to `HandlerPatch.create_artists`, so can be left out.